protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -924,11 +924,40 @@ service TradingDataService {
 
 // -- Pagination --
 
-// Cursor information for cursor based pagination requests
+// Pagination information for cursor based pagination.
+//
+// All data returned from the API is ordered in some well defined manner. The details of which
+// columns and in which order the sorting is performed depend on which API is being endpoint is 
+// being called, but usually the timestamp of the block in which it data was last updated is the
+// primary sort column.
```

Comment:
`depend on which API is being endpoint is being called`: not sure this is making sense.
`the block in which it data`: this too.